[2025-05-21T09:36:25.089+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: yellow_taxi_batch_pipeline.transform_taxi_data manual__2025-05-21T09:35:11.589791+00:00 [queued]>
[2025-05-21T09:36:25.097+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: yellow_taxi_batch_pipeline.transform_taxi_data manual__2025-05-21T09:35:11.589791+00:00 [queued]>
[2025-05-21T09:36:25.098+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 2
[2025-05-21T09:36:25.115+0000] {taskinstance.py:1380} INFO - Executing <Task(BashOperator): transform_taxi_data> on 2025-05-21 09:35:11.589791+00:00
[2025-05-21T09:36:25.122+0000] {standard_task_runner.py:57} INFO - Started process 658 to run task
[2025-05-21T09:36:25.127+0000] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'yellow_taxi_batch_pipeline', 'transform_taxi_data', 'manual__2025-05-21T09:35:11.589791+00:00', '--job-id', '69', '--raw', '--subdir', 'DAGS_FOLDER/taxi_batch_dag.py', '--cfg-path', '/tmp/tmpfxse1pya']
[2025-05-21T09:36:25.135+0000] {standard_task_runner.py:85} INFO - Job 69: Subtask transform_taxi_data
[2025-05-21T09:36:25.162+0000] {logging_mixin.py:151} WARNING - /home/airflow/.local/lib/python3.8/site-packages/airflow/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-05-21T09:36:25.221+0000] {task_command.py:415} INFO - Running <TaskInstance: yellow_taxi_batch_pipeline.transform_taxi_data manual__2025-05-21T09:35:11.589791+00:00 [running]> on host e235d42916dc
[2025-05-21T09:36:25.399+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='yellow_taxi_batch_pipeline' AIRFLOW_CTX_TASK_ID='transform_taxi_data' AIRFLOW_CTX_EXECUTION_DATE='2025-05-21T09:35:11.589791+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-21T09:35:11.589791+00:00'
[2025-05-21T09:36:25.401+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-05-21T09:36:25.402+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', '\n    spark-submit         --master spark://spark-master:7077         --jars /opt/airflow/dags/jars/***ql-42.5.0.jar         --packages org.apache.hadoop:hadoop-aws:3.3.1         /opt/airflow/dags/scripts/taxi_transform.py\n    ']
[2025-05-21T09:36:25.412+0000] {subprocess.py:86} INFO - Output:
[2025-05-21T09:36:25.420+0000] {subprocess.py:93} INFO - /opt/spark/bin/load-spark-env.sh: line 68: ps: command not found
[2025-05-21T09:36:27.879+0000] {subprocess.py:93} INFO - :: loading settings :: url = jar:file:/opt/spark/jars/ivy-2.5.0.jar!/org/apache/ivy/core/settings/ivysettings.xml
[2025-05-21T09:36:28.050+0000] {subprocess.py:93} INFO - Ivy Default Cache set to: /home/airflow/.ivy2/cache
[2025-05-21T09:36:28.051+0000] {subprocess.py:93} INFO - The jars for the packages stored in: /home/airflow/.ivy2/jars
[2025-05-21T09:36:28.060+0000] {subprocess.py:93} INFO - org.apache.hadoop#hadoop-aws added as a dependency
[2025-05-21T09:36:28.061+0000] {subprocess.py:93} INFO - :: resolving dependencies :: org.apache.spark#spark-submit-parent-946f32fa-7047-4ee4-8306-221cf02a3f5e;1.0
[2025-05-21T09:36:28.062+0000] {subprocess.py:93} INFO - 	confs: [default]
[2025-05-21T09:36:30.279+0000] {subprocess.py:93} INFO - 	found org.apache.hadoop#hadoop-aws;3.3.1 in central
[2025-05-21T09:36:30.954+0000] {subprocess.py:93} INFO - 	found com.amazonaws#aws-java-sdk-bundle;1.11.901 in central
[2025-05-21T09:36:32.263+0000] {subprocess.py:93} INFO - 	found org.wildfly.openssl#wildfly-openssl;1.0.7.Final in central
[2025-05-21T09:36:32.290+0000] {subprocess.py:93} INFO - downloading https://repo1.maven.org/maven2/org/apache/hadoop/hadoop-aws/3.3.1/hadoop-aws-3.3.1.jar ...
[2025-05-21T09:36:32.389+0000] {subprocess.py:93} INFO - 	[SUCCESSFUL ] org.apache.hadoop#hadoop-aws;3.3.1!hadoop-aws.jar (113ms)
[2025-05-21T09:36:32.406+0000] {subprocess.py:93} INFO - downloading https://repo1.maven.org/maven2/com/amazonaws/aws-java-sdk-bundle/1.11.901/aws-java-sdk-bundle-1.11.901.jar ...
[2025-05-21T09:36:51.913+0000] {subprocess.py:93} INFO - 	[SUCCESSFUL ] com.amazonaws#aws-java-sdk-bundle;1.11.901!aws-java-sdk-bundle.jar (19520ms)
[2025-05-21T09:36:51.930+0000] {subprocess.py:93} INFO - downloading https://repo1.maven.org/maven2/org/wildfly/openssl/wildfly-openssl/1.0.7.Final/wildfly-openssl-1.0.7.Final.jar ...
[2025-05-21T09:36:51.994+0000] {subprocess.py:93} INFO - 	[SUCCESSFUL ] org.wildfly.openssl#wildfly-openssl;1.0.7.Final!wildfly-openssl.jar (78ms)
[2025-05-21T09:36:51.997+0000] {subprocess.py:93} INFO - :: resolution report :: resolve 4213ms :: artifacts dl 19722ms
[2025-05-21T09:36:51.998+0000] {subprocess.py:93} INFO - 	:: modules in use:
[2025-05-21T09:36:52.004+0000] {subprocess.py:93} INFO - 	com.amazonaws#aws-java-sdk-bundle;1.11.901 from central in [default]
[2025-05-21T09:36:52.004+0000] {subprocess.py:93} INFO - 	org.apache.hadoop#hadoop-aws;3.3.1 from central in [default]
[2025-05-21T09:36:52.005+0000] {subprocess.py:93} INFO - 	org.wildfly.openssl#wildfly-openssl;1.0.7.Final from central in [default]
[2025-05-21T09:36:52.005+0000] {subprocess.py:93} INFO - 	---------------------------------------------------------------------
[2025-05-21T09:36:52.007+0000] {subprocess.py:93} INFO - 	|                  |            modules            ||   artifacts   |
[2025-05-21T09:36:52.007+0000] {subprocess.py:93} INFO - 	|       conf       | number| search|dwnlded|evicted|| number|dwnlded|
[2025-05-21T09:36:52.008+0000] {subprocess.py:93} INFO - 	---------------------------------------------------------------------
[2025-05-21T09:36:52.008+0000] {subprocess.py:93} INFO - 	|      default     |   3   |   3   |   3   |   0   ||   3   |   3   |
[2025-05-21T09:36:52.009+0000] {subprocess.py:93} INFO - 	---------------------------------------------------------------------
[2025-05-21T09:36:52.009+0000] {subprocess.py:93} INFO - :: retrieving :: org.apache.spark#spark-submit-parent-946f32fa-7047-4ee4-8306-221cf02a3f5e
[2025-05-21T09:36:52.010+0000] {subprocess.py:93} INFO - 	confs: [default]
[2025-05-21T09:36:52.323+0000] {subprocess.py:93} INFO - 	3 artifacts copied, 0 already retrieved (189078kB/317ms)
[2025-05-21T09:36:52.689+0000] {subprocess.py:93} INFO - 25/05/21 09:36:52 WARN NativeCodeLoader: Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
[2025-05-21T09:36:53.040+0000] {subprocess.py:93} INFO - 25/05/21 09:36:53 WARN DependencyUtils: Local jar /opt/airflow/dags/jars/***ql-42.5.0.jar does not exist, skipping.
[2025-05-21T09:36:55.373+0000] {subprocess.py:93} INFO - 25/05/21 09:36:55 INFO SparkContext: Running Spark version 3.3.1
[2025-05-21T09:36:55.416+0000] {subprocess.py:93} INFO - 25/05/21 09:36:55 INFO ResourceUtils: ==============================================================
[2025-05-21T09:36:55.417+0000] {subprocess.py:93} INFO - 25/05/21 09:36:55 INFO ResourceUtils: No custom resources configured for spark.driver.
[2025-05-21T09:36:55.417+0000] {subprocess.py:93} INFO - 25/05/21 09:36:55 INFO ResourceUtils: ==============================================================
[2025-05-21T09:36:55.418+0000] {subprocess.py:93} INFO - 25/05/21 09:36:55 INFO SparkContext: Submitted application: YellowTaxiTransform
[2025-05-21T09:36:55.494+0000] {subprocess.py:93} INFO - 25/05/21 09:36:55 INFO ResourceProfile: Default ResourceProfile created, executor resources: Map(cores -> name: cores, amount: 1, script: , vendor: , memory -> name: memory, amount: 1024, script: , vendor: , offHeap -> name: offHeap, amount: 0, script: , vendor: ), task resources: Map(cpus -> name: cpus, amount: 1.0)
[2025-05-21T09:36:55.530+0000] {subprocess.py:93} INFO - 25/05/21 09:36:55 INFO ResourceProfile: Limiting resource is cpu
[2025-05-21T09:36:55.531+0000] {subprocess.py:93} INFO - 25/05/21 09:36:55 INFO ResourceProfileManager: Added ResourceProfile id: 0
[2025-05-21T09:36:55.626+0000] {subprocess.py:93} INFO - 25/05/21 09:36:55 INFO SecurityManager: Changing view acls to: airflow
[2025-05-21T09:36:55.627+0000] {subprocess.py:93} INFO - 25/05/21 09:36:55 INFO SecurityManager: Changing modify acls to: airflow
[2025-05-21T09:36:55.628+0000] {subprocess.py:93} INFO - 25/05/21 09:36:55 INFO SecurityManager: Changing view acls groups to:
[2025-05-21T09:36:55.628+0000] {subprocess.py:93} INFO - 25/05/21 09:36:55 INFO SecurityManager: Changing modify acls groups to:
[2025-05-21T09:36:55.629+0000] {subprocess.py:93} INFO - 25/05/21 09:36:55 INFO SecurityManager: SecurityManager: authentication disabled; ui acls disabled; users  with view permissions: Set(airflow); groups with view permissions: Set(); users  with modify permissions: Set(airflow); groups with modify permissions: Set()
[2025-05-21T09:36:56.433+0000] {subprocess.py:93} INFO - 25/05/21 09:36:56 INFO Utils: Successfully started service 'sparkDriver' on port 38123.
[2025-05-21T09:36:56.573+0000] {subprocess.py:93} INFO - 25/05/21 09:36:56 INFO SparkEnv: Registering MapOutputTracker
[2025-05-21T09:36:56.752+0000] {subprocess.py:93} INFO - 25/05/21 09:36:56 INFO SparkEnv: Registering BlockManagerMaster
[2025-05-21T09:36:56.821+0000] {subprocess.py:93} INFO - 25/05/21 09:36:56 INFO BlockManagerMasterEndpoint: Using org.apache.spark.storage.DefaultTopologyMapper for getting topology information
[2025-05-21T09:36:56.822+0000] {subprocess.py:93} INFO - 25/05/21 09:36:56 INFO BlockManagerMasterEndpoint: BlockManagerMasterEndpoint up
[2025-05-21T09:36:56.836+0000] {subprocess.py:93} INFO - 25/05/21 09:36:56 INFO SparkEnv: Registering BlockManagerMasterHeartbeat
[2025-05-21T09:36:56.912+0000] {subprocess.py:93} INFO - 25/05/21 09:36:56 INFO DiskBlockManager: Created local directory at /tmp/blockmgr-72fe01c8-a4f9-4fc0-831a-571badd2557f
[2025-05-21T09:36:56.996+0000] {subprocess.py:93} INFO - 25/05/21 09:36:56 INFO MemoryStore: MemoryStore started with capacity 434.4 MiB
[2025-05-21T09:36:57.042+0000] {subprocess.py:93} INFO - 25/05/21 09:36:57 INFO SparkEnv: Registering OutputCommitCoordinator
[2025-05-21T09:36:57.481+0000] {subprocess.py:93} INFO - 25/05/21 09:36:57 INFO Utils: Successfully started service 'SparkUI' on port 4040.
[2025-05-21T09:36:57.575+0000] {subprocess.py:93} INFO - 25/05/21 09:36:57 ERROR SparkContext: Failed to add /opt/spark/jars/***ql-42.5.0.jar to Spark environment
[2025-05-21T09:36:57.576+0000] {subprocess.py:93} INFO - java.io.FileNotFoundException: Jar /opt/spark/jars/***ql-42.5.0.jar not found
[2025-05-21T09:36:57.577+0000] {subprocess.py:93} INFO - 	at org.apache.spark.SparkContext.addLocalJarFile$1(SparkContext.scala:1949)
[2025-05-21T09:36:57.579+0000] {subprocess.py:93} INFO - 	at org.apache.spark.SparkContext.addJar(SparkContext.scala:2004)
[2025-05-21T09:36:57.580+0000] {subprocess.py:93} INFO - 	at org.apache.spark.SparkContext.$anonfun$new$12(SparkContext.scala:507)
[2025-05-21T09:36:57.580+0000] {subprocess.py:93} INFO - 	at org.apache.spark.SparkContext.$anonfun$new$12$adapted(SparkContext.scala:507)
[2025-05-21T09:36:57.581+0000] {subprocess.py:93} INFO - 	at scala.collection.mutable.ResizableArray.foreach(ResizableArray.scala:62)
[2025-05-21T09:36:57.581+0000] {subprocess.py:93} INFO - 	at scala.collection.mutable.ResizableArray.foreach$(ResizableArray.scala:55)
[2025-05-21T09:36:57.582+0000] {subprocess.py:93} INFO - 	at scala.collection.mutable.ArrayBuffer.foreach(ArrayBuffer.scala:49)
[2025-05-21T09:36:57.582+0000] {subprocess.py:93} INFO - 	at org.apache.spark.SparkContext.<init>(SparkContext.scala:507)
[2025-05-21T09:36:57.582+0000] {subprocess.py:93} INFO - 	at org.apache.spark.api.java.JavaSparkContext.<init>(JavaSparkContext.scala:58)
[2025-05-21T09:36:57.583+0000] {subprocess.py:93} INFO - 	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
[2025-05-21T09:36:57.583+0000] {subprocess.py:93} INFO - 	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
[2025-05-21T09:36:57.584+0000] {subprocess.py:93} INFO - 	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
[2025-05-21T09:36:57.584+0000] {subprocess.py:93} INFO - 	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:490)
[2025-05-21T09:36:57.585+0000] {subprocess.py:93} INFO - 	at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:247)
[2025-05-21T09:36:57.585+0000] {subprocess.py:93} INFO - 	at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:357)
[2025-05-21T09:36:57.586+0000] {subprocess.py:93} INFO - 	at py4j.Gateway.invoke(Gateway.java:238)
[2025-05-21T09:36:57.586+0000] {subprocess.py:93} INFO - 	at py4j.commands.ConstructorCommand.invokeConstructor(ConstructorCommand.java:80)
[2025-05-21T09:36:57.587+0000] {subprocess.py:93} INFO - 	at py4j.commands.ConstructorCommand.execute(ConstructorCommand.java:69)
[2025-05-21T09:36:57.587+0000] {subprocess.py:93} INFO - 	at py4j.ClientServerConnection.waitForCommands(ClientServerConnection.java:182)
[2025-05-21T09:36:57.587+0000] {subprocess.py:93} INFO - 	at py4j.ClientServerConnection.run(ClientServerConnection.java:106)
[2025-05-21T09:36:57.588+0000] {subprocess.py:93} INFO - 	at java.base/java.lang.Thread.run(Thread.java:829)
[2025-05-21T09:36:57.589+0000] {subprocess.py:93} INFO - 25/05/21 09:36:57 INFO SparkContext: Added file file:///home/airflow/.ivy2/jars/org.apache.hadoop_hadoop-aws-3.3.1.jar at spark://e235d42916dc:38123/files/org.apache.hadoop_hadoop-aws-3.3.1.jar with timestamp 1747820215342
[2025-05-21T09:36:57.590+0000] {subprocess.py:93} INFO - 25/05/21 09:36:57 INFO Utils: Copying /home/airflow/.ivy2/jars/org.apache.hadoop_hadoop-aws-3.3.1.jar to /tmp/spark-fc43daf2-67dc-48fd-b087-d3ad9c9aa318/userFiles-2650ffb8-b718-45c5-a313-bd3fcc604f9d/org.apache.hadoop_hadoop-aws-3.3.1.jar
[2025-05-21T09:36:57.604+0000] {subprocess.py:93} INFO - 25/05/21 09:36:57 INFO SparkContext: Added file file:///home/airflow/.ivy2/jars/com.amazonaws_aws-java-sdk-bundle-1.11.901.jar at spark://e235d42916dc:38123/files/com.amazonaws_aws-java-sdk-bundle-1.11.901.jar with timestamp 1747820215342
[2025-05-21T09:36:57.605+0000] {subprocess.py:93} INFO - 25/05/21 09:36:57 INFO Utils: Copying /home/airflow/.ivy2/jars/com.amazonaws_aws-java-sdk-bundle-1.11.901.jar to /tmp/spark-fc43daf2-67dc-48fd-b087-d3ad9c9aa318/userFiles-2650ffb8-b718-45c5-a313-bd3fcc604f9d/com.amazonaws_aws-java-sdk-bundle-1.11.901.jar
[2025-05-21T09:36:58.041+0000] {subprocess.py:93} INFO - 25/05/21 09:36:58 INFO SparkContext: Added file file:///home/airflow/.ivy2/jars/org.wildfly.openssl_wildfly-openssl-1.0.7.Final.jar at spark://e235d42916dc:38123/files/org.wildfly.openssl_wildfly-openssl-1.0.7.Final.jar with timestamp 1747820215342
[2025-05-21T09:36:58.043+0000] {subprocess.py:93} INFO - 25/05/21 09:36:58 INFO Utils: Copying /home/airflow/.ivy2/jars/org.wildfly.openssl_wildfly-openssl-1.0.7.Final.jar to /tmp/spark-fc43daf2-67dc-48fd-b087-d3ad9c9aa318/userFiles-2650ffb8-b718-45c5-a313-bd3fcc604f9d/org.wildfly.openssl_wildfly-openssl-1.0.7.Final.jar
[2025-05-21T09:36:58.278+0000] {subprocess.py:93} INFO - 25/05/21 09:36:58 INFO StandaloneAppClient$ClientEndpoint: Connecting to master spark://spark-master:7077...
[2025-05-21T09:36:58.401+0000] {subprocess.py:93} INFO - 25/05/21 09:36:58 INFO TransportClientFactory: Successfully created connection to spark-master/172.18.0.3:7077 after 95 ms (0 ms spent in bootstraps)
[2025-05-21T09:36:58.537+0000] {subprocess.py:93} INFO - 25/05/21 09:36:58 INFO StandaloneSchedulerBackend: Connected to Spark cluster with app ID app-20250521093658-0002
[2025-05-21T09:36:58.548+0000] {subprocess.py:93} INFO - 25/05/21 09:36:58 INFO StandaloneAppClient$ClientEndpoint: Executor added: app-20250521093658-0002/0 on worker-20250521093358-172.18.0.5-36249 (172.18.0.5:36249) with 2 core(s)
[2025-05-21T09:36:58.558+0000] {subprocess.py:93} INFO - 25/05/21 09:36:58 INFO StandaloneSchedulerBackend: Granted executor ID app-20250521093658-0002/0 on hostPort 172.18.0.5:36249 with 2 core(s), 1024.0 MiB RAM
[2025-05-21T09:36:58.562+0000] {subprocess.py:93} INFO - 25/05/21 09:36:58 INFO Utils: Successfully started service 'org.apache.spark.network.netty.NettyBlockTransferService' on port 36071.
[2025-05-21T09:36:58.562+0000] {subprocess.py:93} INFO - 25/05/21 09:36:58 INFO NettyBlockTransferService: Server created on e235d42916dc:36071
[2025-05-21T09:36:58.565+0000] {subprocess.py:93} INFO - 25/05/21 09:36:58 INFO BlockManager: Using org.apache.spark.storage.RandomBlockReplicationPolicy for block replication policy
[2025-05-21T09:36:58.591+0000] {subprocess.py:93} INFO - 25/05/21 09:36:58 INFO BlockManagerMaster: Registering BlockManager BlockManagerId(driver, e235d42916dc, 36071, None)
[2025-05-21T09:36:58.599+0000] {subprocess.py:93} INFO - 25/05/21 09:36:58 INFO BlockManagerMasterEndpoint: Registering block manager e235d42916dc:36071 with 434.4 MiB RAM, BlockManagerId(driver, e235d42916dc, 36071, None)
[2025-05-21T09:36:58.602+0000] {subprocess.py:93} INFO - 25/05/21 09:36:58 INFO BlockManagerMaster: Registered BlockManager BlockManagerId(driver, e235d42916dc, 36071, None)
[2025-05-21T09:36:58.605+0000] {subprocess.py:93} INFO - 25/05/21 09:36:58 INFO BlockManager: Initialized BlockManager: BlockManagerId(driver, e235d42916dc, 36071, None)
[2025-05-21T09:36:58.703+0000] {subprocess.py:93} INFO - 25/05/21 09:36:58 INFO StandaloneAppClient$ClientEndpoint: Executor updated: app-20250521093658-0002/0 is now RUNNING
[2025-05-21T09:36:59.149+0000] {subprocess.py:93} INFO - 25/05/21 09:36:59 INFO StandaloneSchedulerBackend: SchedulerBackend is ready for scheduling beginning after reached minRegisteredResourcesRatio: 0.0
[2025-05-21T09:36:59.744+0000] {subprocess.py:93} INFO - 25/05/21 09:36:59 INFO SharedState: Setting hive.metastore.warehouse.dir ('null') to the value of spark.sql.warehouse.dir.
[2025-05-21T09:36:59.748+0000] {subprocess.py:93} INFO - 25/05/21 09:36:59 INFO SharedState: Warehouse path is 'file:/tmp/airflowtmp50jsrt2n/spark-warehouse'.
[2025-05-21T09:37:01.938+0000] {subprocess.py:93} INFO - 2025-05-21 09:37:01,936 - INFO - Loading Yellow Taxi data from MinIO
[2025-05-21T09:37:02.513+0000] {subprocess.py:93} INFO - 25/05/21 09:37:02 WARN MetricsConfig: Cannot locate configuration: tried hadoop-metrics2-s3a-file-system.properties,hadoop-metrics2.properties
[2025-05-21T09:37:02.570+0000] {subprocess.py:93} INFO - 25/05/21 09:37:02 INFO MetricsSystemImpl: Scheduled Metric snapshot period at 10 second(s).
[2025-05-21T09:37:02.583+0000] {subprocess.py:93} INFO - 25/05/21 09:37:02 INFO MetricsSystemImpl: s3a-file-system metrics system started
[2025-05-21T09:37:05.377+0000] {subprocess.py:93} INFO - 25/05/21 09:37:05 INFO CoarseGrainedSchedulerBackend$DriverEndpoint: Registered executor NettyRpcEndpointRef(spark-client://Executor) (172.18.0.5:49440) with ID 0,  ResourceProfileId 0
[2025-05-21T09:37:05.682+0000] {subprocess.py:93} INFO - 25/05/21 09:37:05 INFO BlockManagerMasterEndpoint: Registering block manager 172.18.0.5:45841 with 366.3 MiB RAM, BlockManagerId(0, 172.18.0.5, 45841, None)
[2025-05-21T09:37:06.310+0000] {subprocess.py:93} INFO - 25/05/21 09:37:06 WARN FileStreamSink: Assume no metadata directory. Error while looking for metadata directory in the path: s3a://nyc-taxi-data/yellow_taxi/yellow_tripdata_2022-*.parquet.
[2025-05-21T09:37:06.327+0000] {subprocess.py:93} INFO - java.nio.file.AccessDeniedException: s3a://nyc-taxi-data/yellow_taxi/yellow_tripdata_2022-*.parquet: org.apache.hadoop.fs.s3a.auth.NoAuthWithAWSException: No AWS Credentials provided by TemporaryAWSCredentialsProvider SimpleAWSCredentialsProvider EnvironmentVariableCredentialsProvider IAMInstanceCredentialsProvider : com.amazonaws.SdkClientException: Unable to load AWS credentials from environment variables (AWS_ACCESS_KEY_ID (or AWS_ACCESS_KEY) and AWS_SECRET_KEY (or AWS_SECRET_ACCESS_KEY))
[2025-05-21T09:37:06.358+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AUtils.translateException(S3AUtils.java:206)
[2025-05-21T09:37:06.374+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AUtils.translateException(S3AUtils.java:170)
[2025-05-21T09:37:06.397+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.s3GetFileStatus(S3AFileSystem.java:3351)
[2025-05-21T09:37:06.414+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.innerGetFileStatus(S3AFileSystem.java:3185)
[2025-05-21T09:37:06.467+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.isDirectory(S3AFileSystem.java:4277)
[2025-05-21T09:37:06.488+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.streaming.FileStreamSink$.hasMetadata(FileStreamSink.scala:54)
[2025-05-21T09:37:06.512+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.DataSource.resolveRelation(DataSource.scala:370)
[2025-05-21T09:37:06.546+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.DataFrameReader.loadV1Source(DataFrameReader.scala:228)
[2025-05-21T09:37:06.558+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.DataFrameReader.$anonfun$load$2(DataFrameReader.scala:210)
[2025-05-21T09:37:06.587+0000] {subprocess.py:93} INFO - 	at scala.Option.getOrElse(Option.scala:189)
[2025-05-21T09:37:06.603+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.DataFrameReader.load(DataFrameReader.scala:210)
[2025-05-21T09:37:06.627+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.DataFrameReader.parquet(DataFrameReader.scala:562)
[2025-05-21T09:37:06.652+0000] {subprocess.py:93} INFO - 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[2025-05-21T09:37:06.656+0000] {subprocess.py:93} INFO - 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
[2025-05-21T09:37:06.668+0000] {subprocess.py:93} INFO - 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[2025-05-21T09:37:06.670+0000] {subprocess.py:93} INFO - 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
[2025-05-21T09:37:06.672+0000] {subprocess.py:93} INFO - 	at py4j.reflection.MethodInvoker.invoke(MethodInvoker.java:244)
[2025-05-21T09:37:06.688+0000] {subprocess.py:93} INFO - 	at py4j.reflection.ReflectionEngine.invoke(ReflectionEngine.java:357)
[2025-05-21T09:37:06.691+0000] {subprocess.py:93} INFO - 	at py4j.Gateway.invoke(Gateway.java:282)
[2025-05-21T09:37:06.708+0000] {subprocess.py:93} INFO - 	at py4j.commands.AbstractCommand.invokeMethod(AbstractCommand.java:132)
[2025-05-21T09:37:06.724+0000] {subprocess.py:93} INFO - 	at py4j.commands.CallCommand.execute(CallCommand.java:79)
[2025-05-21T09:37:06.749+0000] {subprocess.py:93} INFO - 	at py4j.ClientServerConnection.waitForCommands(ClientServerConnection.java:182)
[2025-05-21T09:37:06.751+0000] {subprocess.py:93} INFO - 	at py4j.ClientServerConnection.run(ClientServerConnection.java:106)
[2025-05-21T09:37:06.755+0000] {subprocess.py:93} INFO - 	at java.base/java.lang.Thread.run(Thread.java:829)
[2025-05-21T09:37:06.766+0000] {subprocess.py:93} INFO - Caused by: org.apache.hadoop.fs.s3a.auth.NoAuthWithAWSException: No AWS Credentials provided by TemporaryAWSCredentialsProvider SimpleAWSCredentialsProvider EnvironmentVariableCredentialsProvider IAMInstanceCredentialsProvider : com.amazonaws.SdkClientException: Unable to load AWS credentials from environment variables (AWS_ACCESS_KEY_ID (or AWS_ACCESS_KEY) and AWS_SECRET_KEY (or AWS_SECRET_ACCESS_KEY))
[2025-05-21T09:37:06.768+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.AWSCredentialProviderList.getCredentials(AWSCredentialProviderList.java:216)
[2025-05-21T09:37:06.769+0000] {subprocess.py:93} INFO - 	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.getCredentialsFromContext(AmazonHttpClient.java:1257)
[2025-05-21T09:37:06.770+0000] {subprocess.py:93} INFO - 	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.runBeforeRequestHandlers(AmazonHttpClient.java:833)
[2025-05-21T09:37:06.772+0000] {subprocess.py:93} INFO - 	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.doExecute(AmazonHttpClient.java:783)
[2025-05-21T09:37:06.772+0000] {subprocess.py:93} INFO - 	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.executeWithTimer(AmazonHttpClient.java:770)
[2025-05-21T09:37:06.774+0000] {subprocess.py:93} INFO - 	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.execute(AmazonHttpClient.java:744)
[2025-05-21T09:37:06.775+0000] {subprocess.py:93} INFO - 	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.access$500(AmazonHttpClient.java:704)
[2025-05-21T09:37:06.776+0000] {subprocess.py:93} INFO - 	at com.amazonaws.http.AmazonHttpClient$RequestExecutionBuilderImpl.execute(AmazonHttpClient.java:686)
[2025-05-21T09:37:06.785+0000] {subprocess.py:93} INFO - 	at com.amazonaws.http.AmazonHttpClient.execute(AmazonHttpClient.java:550)
[2025-05-21T09:37:06.786+0000] {subprocess.py:93} INFO - 	at com.amazonaws.http.AmazonHttpClient.execute(AmazonHttpClient.java:530)
[2025-05-21T09:37:06.787+0000] {subprocess.py:93} INFO - 	at com.amazonaws.services.s3.AmazonS3Client.invoke(AmazonS3Client.java:5227)
[2025-05-21T09:37:06.787+0000] {subprocess.py:93} INFO - 	at com.amazonaws.services.s3.AmazonS3Client.getBucketRegionViaHeadRequest(AmazonS3Client.java:6189)
[2025-05-21T09:37:06.797+0000] {subprocess.py:93} INFO - 	at com.amazonaws.services.s3.AmazonS3Client.fetchRegionFromCache(AmazonS3Client.java:6162)
[2025-05-21T09:37:06.798+0000] {subprocess.py:93} INFO - 	at com.amazonaws.services.s3.AmazonS3Client.invoke(AmazonS3Client.java:5211)
[2025-05-21T09:37:06.801+0000] {subprocess.py:93} INFO - 	at com.amazonaws.services.s3.AmazonS3Client.invoke(AmazonS3Client.java:5173)
[2025-05-21T09:37:06.808+0000] {subprocess.py:93} INFO - 	at com.amazonaws.services.s3.AmazonS3Client.invoke(AmazonS3Client.java:5167)
[2025-05-21T09:37:06.820+0000] {subprocess.py:93} INFO - 	at com.amazonaws.services.s3.AmazonS3Client.listObjectsV2(AmazonS3Client.java:963)
[2025-05-21T09:37:06.828+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.lambda$listObjects$7(S3AFileSystem.java:2116)
[2025-05-21T09:37:06.834+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.statistics.impl.IOStatisticsBinding.lambda$trackDurationOfOperation$5(IOStatisticsBinding.java:499)
[2025-05-21T09:37:06.836+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.Invoker.retryUntranslated(Invoker.java:412)
[2025-05-21T09:37:06.849+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.Invoker.retryUntranslated(Invoker.java:375)
[2025-05-21T09:37:06.853+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.listObjects(S3AFileSystem.java:2107)
[2025-05-21T09:37:06.861+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.s3GetFileStatus(S3AFileSystem.java:3322)
[2025-05-21T09:37:06.862+0000] {subprocess.py:93} INFO - 	... 21 more
[2025-05-21T09:37:06.875+0000] {subprocess.py:93} INFO - Caused by: com.amazonaws.SdkClientException: Unable to load AWS credentials from environment variables (AWS_ACCESS_KEY_ID (or AWS_ACCESS_KEY) and AWS_SECRET_KEY (or AWS_SECRET_ACCESS_KEY))
[2025-05-21T09:37:06.877+0000] {subprocess.py:93} INFO - 	at com.amazonaws.auth.EnvironmentVariableCredentialsProvider.getCredentials(EnvironmentVariableCredentialsProvider.java:50)
[2025-05-21T09:37:06.878+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.AWSCredentialProviderList.getCredentials(AWSCredentialProviderList.java:177)
[2025-05-21T09:37:06.880+0000] {subprocess.py:93} INFO - 	... 43 more
[2025-05-21T09:37:08.816+0000] {subprocess.py:93} INFO - 2025-05-21 09:37:08,813 - ERROR - ETL process failed: An error occurred while calling o31.parquet.
[2025-05-21T09:37:08.822+0000] {subprocess.py:93} INFO - : java.nio.file.AccessDeniedException: s3a://nyc-taxi-data/yellow_taxi: org.apache.hadoop.fs.s3a.auth.NoAuthWithAWSException: No AWS Credentials provided by TemporaryAWSCredentialsProvider SimpleAWSCredentialsProvider EnvironmentVariableCredentialsProvider IAMInstanceCredentialsProvider : com.amazonaws.SdkClientException: Unable to load AWS credentials from environment variables (AWS_ACCESS_KEY_ID (or AWS_ACCESS_KEY) and AWS_SECRET_KEY (or AWS_SECRET_ACCESS_KEY))
[2025-05-21T09:37:08.839+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AUtils.translateException(S3AUtils.java:206)
[2025-05-21T09:37:08.857+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.Invoker.once(Invoker.java:117)
[2025-05-21T09:37:08.861+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.listStatus(S3AFileSystem.java:2810)
[2025-05-21T09:37:08.877+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.Globber.listStatus(Globber.java:128)
[2025-05-21T09:37:08.884+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.Globber.doGlob(Globber.java:291)
[2025-05-21T09:37:08.929+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.Globber.glob(Globber.java:202)
[2025-05-21T09:37:08.953+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.globStatus(S3AFileSystem.java:4253)
[2025-05-21T09:37:08.964+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.globStatus(S3AFileSystem.java:4233)
[2025-05-21T09:37:08.965+0000] {subprocess.py:93} INFO - 	at org.apache.spark.deploy.SparkHadoopUtil.globPath(SparkHadoopUtil.scala:253)
[2025-05-21T09:37:08.973+0000] {subprocess.py:93} INFO - 	at org.apache.spark.sql.execution.datasources.DataSource$.$anonfun$checkAndGlobPathIfNecessary$3(DataSource.scala:765)
[2025-05-21T09:37:08.977+0000] {subprocess.py:93} INFO - 	at org.apache.spark.util.ThreadUtils$.$anonfun$parmap$2(ThreadUtils.scala:372)
[2025-05-21T09:37:08.978+0000] {subprocess.py:93} INFO - 	at scala.concurrent.Future$.$anonfun$apply$1(Future.scala:659)
[2025-05-21T09:37:08.987+0000] {subprocess.py:93} INFO - 	at scala.util.Success.$anonfun$map$1(Try.scala:255)
[2025-05-21T09:37:08.989+0000] {subprocess.py:93} INFO - 	at scala.util.Success.map(Try.scala:213)
[2025-05-21T09:37:08.989+0000] {subprocess.py:93} INFO - 	at scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
[2025-05-21T09:37:08.990+0000] {subprocess.py:93} INFO - 	at scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
[2025-05-21T09:37:08.990+0000] {subprocess.py:93} INFO - 	at scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
[2025-05-21T09:37:08.991+0000] {subprocess.py:93} INFO - 	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
[2025-05-21T09:37:08.996+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ForkJoinTask$RunnableExecuteAction.exec(ForkJoinTask.java:1426)
[2025-05-21T09:37:09.002+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:290)
[2025-05-21T09:37:09.003+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1020)
[2025-05-21T09:37:09.005+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1656)
[2025-05-21T09:37:09.009+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1594)
[2025-05-21T09:37:09.017+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:183)
[2025-05-21T09:37:09.025+0000] {subprocess.py:93} INFO - Caused by: org.apache.hadoop.fs.s3a.auth.NoAuthWithAWSException: No AWS Credentials provided by TemporaryAWSCredentialsProvider SimpleAWSCredentialsProvider EnvironmentVariableCredentialsProvider IAMInstanceCredentialsProvider : com.amazonaws.SdkClientException: Unable to load AWS credentials from environment variables (AWS_ACCESS_KEY_ID (or AWS_ACCESS_KEY) and AWS_SECRET_KEY (or AWS_SECRET_ACCESS_KEY))
[2025-05-21T09:37:09.030+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.AWSCredentialProviderList.getCredentials(AWSCredentialProviderList.java:216)
[2025-05-21T09:37:09.030+0000] {subprocess.py:93} INFO - 	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.getCredentialsFromContext(AmazonHttpClient.java:1257)
[2025-05-21T09:37:09.031+0000] {subprocess.py:93} INFO - 	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.runBeforeRequestHandlers(AmazonHttpClient.java:833)
[2025-05-21T09:37:09.031+0000] {subprocess.py:93} INFO - 	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.doExecute(AmazonHttpClient.java:783)
[2025-05-21T09:37:09.032+0000] {subprocess.py:93} INFO - 	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.executeWithTimer(AmazonHttpClient.java:770)
[2025-05-21T09:37:09.032+0000] {subprocess.py:93} INFO - 	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.execute(AmazonHttpClient.java:744)
[2025-05-21T09:37:09.033+0000] {subprocess.py:93} INFO - 	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.access$500(AmazonHttpClient.java:704)
[2025-05-21T09:37:09.036+0000] {subprocess.py:93} INFO - 	at com.amazonaws.http.AmazonHttpClient$RequestExecutionBuilderImpl.execute(AmazonHttpClient.java:686)
[2025-05-21T09:37:09.045+0000] {subprocess.py:93} INFO - 	at com.amazonaws.http.AmazonHttpClient.execute(AmazonHttpClient.java:550)
[2025-05-21T09:37:09.051+0000] {subprocess.py:93} INFO - 	at com.amazonaws.http.AmazonHttpClient.execute(AmazonHttpClient.java:530)
[2025-05-21T09:37:09.054+0000] {subprocess.py:93} INFO - 	at com.amazonaws.services.s3.AmazonS3Client.invoke(AmazonS3Client.java:5227)
[2025-05-21T09:37:09.063+0000] {subprocess.py:93} INFO - 	at com.amazonaws.services.s3.AmazonS3Client.getBucketRegionViaHeadRequest(AmazonS3Client.java:6189)
[2025-05-21T09:37:09.070+0000] {subprocess.py:93} INFO - 	at com.amazonaws.services.s3.AmazonS3Client.fetchRegionFromCache(AmazonS3Client.java:6162)
[2025-05-21T09:37:09.075+0000] {subprocess.py:93} INFO - 	at com.amazonaws.services.s3.AmazonS3Client.invoke(AmazonS3Client.java:5211)
[2025-05-21T09:37:09.088+0000] {subprocess.py:93} INFO - 	at com.amazonaws.services.s3.AmazonS3Client.invoke(AmazonS3Client.java:5173)
[2025-05-21T09:37:09.089+0000] {subprocess.py:93} INFO - 	at com.amazonaws.services.s3.AmazonS3Client.invoke(AmazonS3Client.java:5167)
[2025-05-21T09:37:09.089+0000] {subprocess.py:93} INFO - 	at com.amazonaws.services.s3.AmazonS3Client.listObjectsV2(AmazonS3Client.java:963)
[2025-05-21T09:37:09.090+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.lambda$listObjects$7(S3AFileSystem.java:2116)
[2025-05-21T09:37:09.091+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.statistics.impl.IOStatisticsBinding.lambda$trackDurationOfOperation$5(IOStatisticsBinding.java:499)
[2025-05-21T09:37:09.092+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.Invoker.retryUntranslated(Invoker.java:412)
[2025-05-21T09:37:09.093+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.Invoker.retryUntranslated(Invoker.java:375)
[2025-05-21T09:37:09.093+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem.listObjects(S3AFileSystem.java:2107)
[2025-05-21T09:37:09.094+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.S3AFileSystem$ListingOperationCallbacksImpl.lambda$listObjectsAsync$0(S3AFileSystem.java:1750)
[2025-05-21T09:37:09.095+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.impl.CallableSupplier.get(CallableSupplier.java:62)
[2025-05-21T09:37:09.096+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1700)
[2025-05-21T09:37:09.096+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
[2025-05-21T09:37:09.097+0000] {subprocess.py:93} INFO - 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
[2025-05-21T09:37:09.097+0000] {subprocess.py:93} INFO - 	at java.base/java.lang.Thread.run(Thread.java:829)
[2025-05-21T09:37:09.098+0000] {subprocess.py:93} INFO - Caused by: com.amazonaws.SdkClientException: Unable to load AWS credentials from environment variables (AWS_ACCESS_KEY_ID (or AWS_ACCESS_KEY) and AWS_SECRET_KEY (or AWS_SECRET_ACCESS_KEY))
[2025-05-21T09:37:09.098+0000] {subprocess.py:93} INFO - 	at com.amazonaws.auth.EnvironmentVariableCredentialsProvider.getCredentials(EnvironmentVariableCredentialsProvider.java:50)
[2025-05-21T09:37:09.099+0000] {subprocess.py:93} INFO - 	at org.apache.hadoop.fs.s3a.AWSCredentialProviderList.getCredentials(AWSCredentialProviderList.java:177)
[2025-05-21T09:37:09.099+0000] {subprocess.py:93} INFO - 	... 27 more
[2025-05-21T09:37:09.100+0000] {subprocess.py:93} INFO - 
[2025-05-21T09:37:09.102+0000] {subprocess.py:93} INFO - 25/05/21 09:37:08 INFO SparkUI: Stopped Spark web UI at http://e235d42916dc:4040
[2025-05-21T09:37:09.102+0000] {subprocess.py:93} INFO - 25/05/21 09:37:08 INFO StandaloneSchedulerBackend: Shutting down all executors
[2025-05-21T09:37:09.103+0000] {subprocess.py:93} INFO - 25/05/21 09:37:08 INFO CoarseGrainedSchedulerBackend$DriverEndpoint: Asking each executor to shut down
[2025-05-21T09:37:09.105+0000] {subprocess.py:93} INFO - 25/05/21 09:37:08 INFO MapOutputTrackerMasterEndpoint: MapOutputTrackerMasterEndpoint stopped!
[2025-05-21T09:37:09.107+0000] {subprocess.py:93} INFO - 25/05/21 09:37:09 INFO MemoryStore: MemoryStore cleared
[2025-05-21T09:37:09.108+0000] {subprocess.py:93} INFO - 25/05/21 09:37:09 INFO BlockManager: BlockManager stopped
[2025-05-21T09:37:09.111+0000] {subprocess.py:93} INFO - 25/05/21 09:37:09 INFO BlockManagerMaster: BlockManagerMaster stopped
[2025-05-21T09:37:09.112+0000] {subprocess.py:93} INFO - 25/05/21 09:37:09 INFO OutputCommitCoordinator$OutputCommitCoordinatorEndpoint: OutputCommitCoordinator stopped!
[2025-05-21T09:37:09.166+0000] {subprocess.py:93} INFO - 25/05/21 09:37:09 INFO SparkContext: Successfully stopped SparkContext
[2025-05-21T09:37:09.404+0000] {subprocess.py:93} INFO - 25/05/21 09:37:09 INFO ShutdownHookManager: Shutdown hook called
[2025-05-21T09:37:09.406+0000] {subprocess.py:93} INFO - 25/05/21 09:37:09 INFO ShutdownHookManager: Deleting directory /tmp/spark-883cc726-39d8-4fb7-8c22-8c056c0959e5
[2025-05-21T09:37:09.427+0000] {subprocess.py:93} INFO - 25/05/21 09:37:09 INFO ShutdownHookManager: Deleting directory /tmp/spark-fc43daf2-67dc-48fd-b087-d3ad9c9aa318/pyspark-639d242e-e584-4d94-a4c7-a07d17a65dca
[2025-05-21T09:37:09.436+0000] {subprocess.py:93} INFO - 25/05/21 09:37:09 INFO ShutdownHookManager: Deleting directory /tmp/spark-fc43daf2-67dc-48fd-b087-d3ad9c9aa318
[2025-05-21T09:37:09.444+0000] {subprocess.py:93} INFO - 25/05/21 09:37:09 INFO MetricsSystemImpl: Stopping s3a-file-system metrics system...
[2025-05-21T09:37:09.445+0000] {subprocess.py:93} INFO - 25/05/21 09:37:09 INFO MetricsSystemImpl: s3a-file-system metrics system stopped.
[2025-05-21T09:37:09.445+0000] {subprocess.py:93} INFO - 25/05/21 09:37:09 INFO MetricsSystemImpl: s3a-file-system metrics system shutdown complete.
[2025-05-21T09:37:09.500+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-05-21T09:37:09.562+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=yellow_taxi_batch_pipeline, task_id=transform_taxi_data, execution_date=20250521T093511, start_date=20250521T093625, end_date=20250521T093709
[2025-05-21T09:37:09.602+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-05-21T09:37:09.642+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
